pkg/tenant/api/v1/api.go

```diff
@@ -277,9 +277,11 @@ type ContactInfo struct {
 	CloudServiceProvider string `json:"cloudServiceProvider"`
 }
 
-// StatCount contains a count for a named statistic which is meant to support a variety
+// StatValue contains a value for a named statistic which is meant to support a variety
 // of statistics endpoints.
-type StatCount struct {
-	Name  string `json:"name"`
-	Count int64  `json:"count"`
+type StatValue struct {
+	Name    string `json:"name"`
+	Value   string `json:"value"`
+	Units   string `json:"units,omitempty"`
+	Percent string `json:"percent,omitempty"`
```

Comment:
Is percent always an independent percentage; e.g. if this is "Storage Used" the value might be "516" and the units are "MiB" and the percent would be something like "23%" e.g. indicating that 23% of the issued capacity is being used? And if there isn't a "total" then percent would be omitted? 

Since we have "units" in the return, is there a reason value and percent aren't numeric (e.g. float64)?

Reply:
The idea is that Name and Value are always required (used for both counts and unit values). The Percent field operates as you described; it only exists for fields which have a "total" (although should we include the total as well?)

I suppose it probably makes more sense to have the value and percent fields be numeric; I was thinking that it didn't make sense for the "count" values to be floats but it might be simpler to have the frontend handle the string formatting depending on the Units field.

Reply:
I agree; I can imagine the situation where they want to round to 2 decimal places or to none in different contexts, so numeric is better. Having a count as a float64 isn't accurate at longer precisions I don't think this endpoint needs extreme precision at all. 

If we have the percent as a float it is possible to compute the total from the value and the percent, so I don't think it necessarily needs to be included at this stage.